docs/en/install.md

```diff
@@ -63,7 +65,58 @@ $ kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/ma
 
 <!-- tabs:end -->
 
-## 2. Get KubeVela
+## 2. Install KubeVela Controller
+
+> Note: This should be prepared by platform builder
+
+These steps will install KubeVela controller and its dependency.
+
+1. Add helm chart repo for KubeVela
+    ```
+    helm repo add kubevela https://kubevelacharts.oss-cn-hangzhou.aliyuncs.com/core
+    ```
+
+2. Update the chart repo
+    ```
+    helm repo update
+    ```
+   
+3. Create Namespace for KubeVela controller
+    ```shell script
+    kubectl create namespace vela-system 
+    ```
+
+    
+
+
+4. Install KubeVela
+    ```shell script
+    helm install -n vela-system kubevela kubevela/vela-core
+    ```
+    By default, it will enable webhook. KubeVela rely on [cert-manager](https://cert-manager.io/docs/)
+    to create certificates for webhook, you must have cert-manager installed.
+    Please refer to [cert-manager installation doc](https://cert-manager.io/docs/installation/kubernetes/) to install
+    if not.
+    
+    You can add an argument `--set useWebhook=false` after the command to disable the webhook,
```

Comment:
Please pay attention to line breaking. Would be better to do this:

```
You can add an argument `--set useWebhook=false` after the command to disable the webhook
if you don't want to rely on cert-manager.
If you just want to have a try this can also work:
```

This would be easier for people who maintain the markdown docs.

Reply:
cool